### PR TITLE
debian/copyright fixes

### PR DESCRIPTION
**Pull request by martinpitt**
These now look like this, actually conforming to the spirit and letter of the [copyright policy](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0):
```
Files: dist/apps/*
Copyright: (c) Facebook, Inc. and its affiliates.
 2015-2016 David Clark
 Copyright (C) 2013 - 2021 Red Hat, Inc.
 Facebook, Inc. and its affiliates.
 Sindre Sorhus
License: LGPL-2.1-or-later and MIT 

Files: dist/base1/*
Copyright: Copyright (C) 2013 - 2021 Red Hat, Inc.
 OpenJS Foundation and other contributors
License: LGPL-2.1-or-later and MIT 
```

[full generated tools/debian/copyright](https://github.com/cockpit-project/cockpit/files/6874038/copyright.txt)

### CODE REVIEW

File: tools/build-debian-copyright [hunk 1]
```diff
@@ -94,20 +97,25 @@ for directory, subdirs, files in os.walk('dist'):
...
+            dist_licenses.setdefault(f'{directory}/*', set()).update(licenses)
 
-            copyrights = '\n '.join(find_patterns(copyright_patterns, comment))
+            copyrights = find_patterns(copyright_patterns, comment)
             if not copyrights:
                 raise SystemError('Did not find any copyrights in:\n%s' % comment)
 
-            paragraph = f'Copyright: {copyrights}\nLicense: {licenses[0]}'
+            # All webpacks also contain our own code
+            copyrights.add("Copyright (C) 2013 - 2021 Red Hat, Inc.")
```
Comment (martinpitt) [inline tools/build-debian-copyright:116]: I don't like the hardcoded "2021". To be completely precise we could iterate over `git grep 'Copyright.*Red Hat'`,  but I suppose for simplicity we could just take the current year?

Comment (martinpitt) [reply tools/build-debian-copyright]: Replaced with current year.

mediocrates

**Comment by martinpitt** [comment]
FTR, lintian is much happier now:
```
Now running lintian cockpit_248-1_source.changes ...
N: 381 hints overridden (381 errors)
Finished running lintian.
gbp:info: Running Postbuild hook
I: cockpit source: unused-license-paragraph-in-dep5-copyright 0bsd (line 295)
P: cockpit source: package-uses-old-debhelper-compat-version 10
```

I'll send a fix for 0bsd in a follow-up commit (after tests are green), but that's just info, not warning any more.

**Review by allisonkarlitskaya — APPROVED** [review]
Thanks for the cleanups

We still need to think about if we want to include notices from non-JS things, and/or encourage our upstreams to mark their comments correctly to be included.

**Pull request merged**